crates/win-api-wrappers/src/utils.rs

```diff
@@ -457,6 +457,69 @@ pub fn expand_environment_path(src: &Path, environment: &HashMap<String, String>
     ))?)
 }
 
+use windows::Win32::Foundation::{BOOL, HMODULE};
+use windows::Win32::Storage::FileSystem::{
+    GetFileVersionInfoSizeW, GetFileVersionInfoW, VerQueryValueW, VS_FIXEDFILEINFO,
+};
+use windows::Win32::System::LibraryLoader::{GetModuleFileNameW, GetModuleHandleW};
+
+pub fn get_exe_version() -> Result<String, anyhow::Error> {
+    // SAFETY: Passing None to GetModuleHandleW requests the handle to the current process main executable module
```

Comment:
nitpick: This is not really a safety comment. A safety comment purpose is to _explain why the unsafe call is safe_. Essentially to make unsafe code easier to audit. In this case, it’s a trivial one.

```suggestion
    // Passing None to GetModuleHandleW requests the handle to the current process main executable module
    // SAFETY: FFI call with no outstanding preconditions.
```